recipe/meta.yaml

```diff
@@ -28,13 +28,21 @@ requirements:
     - numpy >=1.20
     - fasteners
     - numcodecs >=0.10.0
+  run_constrained:
+    # Add `zarr[jupyter]` dependencies to `run_constrained`.
+    # That way the correct versions of these will be installed
+    # when these packages are requested. Otherwise they are not
+    # installed.
+    - notebook
+    - ipytree >=0.2.2
+    - ipywidgets >=8.0.0
```

Comment:
Added these from [the `zarr[jupyter]` dependencies]( https://github.com/zarr-developers/zarr-python/blob/v2.16.0/pyproject.toml#L40-L45 ) to make sure these are properly constrained if installed